exercises/two-bucket/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+/// A struct to hold your results in.
+#[derive(PartialEq, Eq, Debug)]
+pub struct BucketStats<'a> {
+    /// The total number of "moves" it should take to reach the desired number of liters, including
+    /// the first fill.
+    pub moves: u8,
+    /// Which bucket should end up with the desired number of liters? (Either "one" or "two")
+    pub goal_bucket: &'a str,
+    /// How many liters are left in the other bucket?
+    pub other_bucket: u8,
+}
+
+/// Solve the bucket problem
+pub fn solve(_capacity_1: u8,
+             _capacity_2: u8,
+             _goal: u8,
+             _start_bucket: &str) -> BucketStats
+{
+    return BucketStats {
```

Comment:
It's better form to have the body of the stub function be simply `{ unimplemented!() }`, to make it easy for people to see where they should begin work.